pkgs/development/libraries/meshoptimizer/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+let
+  basis_universal = fetchFromGitHub {
+    owner = "zeux";
+    repo = "basis_universal";
+    rev = "8903f6d69849fd782b72a551a4dd04a264434e20";
+    hash = "sha256-o3dCxAAkpMoNkvkM7qD75cPn/obDc/fJ8u7KLPm1G6g=";
+  };
+in stdenv.mkDerivation {
+  pname = "meshoptimizer";
+  version = "unstable-2023-03-22";
+  src = fetchFromGitHub {
+    owner = "zeux";
+    repo = "meshoptimizer";
+    hash = "sha256-OWeptdnKFvTyfkz0sFCpiTI7323GfVE8vb8bNUBnslA=";
+    rev = "49d9222385daf61a9ce75bb4699472408eb3df3e";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  outputs = [ "bin" "dev" "lib" "out" ];
```

Comment:
I have noticed, that the "out" output is empty. I would probably go with this approach, that is documented here: https://github.com/NixOS/nixpkgs/blob/22.11/doc/stdenv/multiple-output.chapter.md#writing-a-split-derivation-sec-multiple-outputs-

So that means changing it to:  `outputs = [ "bin" "dev" "out" ];`